server/src/response.rs

```diff
@@ -1457,6 +1457,7 @@ pub fn make_physical_mode(
         .map(|p| navitia_proto::PhysicalMode {
             uri: Some(format!("{}{}", PREFIX_ID_PHYSICAL_MODE, p.id)),
             name: Some(p.name.clone()),
+            co2_emission_rate: None,
```

Comment:
```suggestion
            co2_emission_rate: p.co2_emission,
```
We might as well fill the field, no ?

Reply:
For sure, with the `rate` on one side, and none on the other, I was not sure if it was the same kind of value.

Reply:
Done in [cb30de9](https://github.com/hove-io/loki/pull/350/commits/cb30de967256c33f4325c738fc2355fa5656807f).